### ISSUE TITLE
Unholy DK diseases cant be dispelled ever please fix major bug you call succubus seduce on a rogue a fuckign critical bug this is far bigger

### ISSUE THREAD

**Issue by sherndy** [dalaranwow/dalaran-wow#5737]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: cant dispell uh dk aids at all if you get coiled once in the past 2 hours

**Expected behaviour**: be able to dispell it as soon as coil ends

**Steps to reproduce the problem**:

1.  be person
2.  get uh dk aids and coil
3.  wait for coil debuff to go away and try to dispell aids

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
We will take a look into this. Closing it for now because of the report's format.
